Tests/Tests/PromiseKit/PromiseTests.swift

```diff
@@ -826,7 +826,7 @@ extension PromiseTests {
   func testConcurrentDispatch1() {
     func next(_ idx: Int) -> Promise<()>? {
       guard idx < 20 else { return nil }
-      return async((), delay: .random(in: 0.01...0.05))
```

Comment:
@gentique I would like to keep it random because that adds a bit of non-determinism in the order in which promises resolve. Can we just lower the range a bit more?